cmd/plugins/juju-metadata/validatetoolsmetadata.go

```diff
@@ -44,12 +44,12 @@ type validateToolsMetadataCommand struct {
 	minor        int
 }
 
-var validateToolsMetadataDoc = `
+var validateAgentsMetadataDoc = `
 validate-agents loads simplestreams metadata and validates the contents by
-looking for agent binaries belonging to the specified series, architecture, for the
-specified cloud. If version is specified, agent binaries matching the exact specified
-version are found. It is also possible to just specify the major (and optionally
-minor) version numbers to search for.
+looking for agent binaries belonging to the specified series, architecture, 
+for the specified cloud. If version is specified, agent binaries matching the 
+exact specified version are found. It is also possible to just specify the
```

Comment:
Maybe `If version is specified, only agent binaries matching that exact version will be considered`?